integration-tests/vault-app/src/test/java/io/quarkus/it/vault/VaultTest.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.it.vault;
+
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.vault.VaultTestExtension;
+import io.restassured.RestAssured;
+
+@QuarkusTest
+@ExtendWith(VaultTestExtension.class)
+@DisabledOnOs(OS.WINDOWS)
```

Comment:
Could you remind me why we do not test on Windows?

Reply:
see https://github.com/quarkusio/quarkus/pull/2897/#discussion_r326310602